.github/workflows/release.yaml

```diff
@@ -27,4 +27,47 @@ jobs:
           args: release --rm-dist
           workdir: ./cmd/manager/ # <-- path to the directory containing the main() function
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  container-images:
+    name: Build and push container images
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_REGISTRY: quay.io
+      DOCKER_REPOSITORY: integreatly/grafana-operator
+      DOCKER_PLATFORMS: linux/amd64,linux/arm64
+    steps:
+      - name: Clone repo and checkout submodules
+        uses: actions/checkout@v2.3.4
+        with:
+          submodules: recursive
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.16"
+      - name: Prepare
+        id: prep
+        run: |
+          VERSION=sha-${GITHUB_SHA::8}
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            VERSION=${GITHUB_REF/refs\/tags\//}
+          fi
+          echo ::set-output name=BUILD_DATE::$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+          echo ::set-output name=VERSION::${VERSION}
+          echo ::set-output name=BUILDARCH::$(go env GOARCH)
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1.3.0
+      - name: Log in to quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.DOCKER_REGISTRY }}
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - name: Build and push multi-arch Docker image
+        uses: docker/build-push-action@v2
+        with:
+          platforms: ${{ env.DOCKER_PLATFORMS }}
+          push: true
+          tags: ${{ env.DOCKER_REGISTRY }}/${{ env.DOCKER_REPOSITORY }}:${{ steps.prep.outputs.VERSION }}
```

Comment:
small nit-pick, can we add an empty line at the end of file? (I'm also guilty of this myself, and I've been scolded by my teammates on multiple occasions :grimacing: )

Reply:
yes, totally – was embarrassed to see that in the diff 😅